### ISSUE TITLE
Cannot access plugin settings

### ISSUE THREAD

**Issue by pimago** [nilsenpaul/craft-complete-cookie-consent#41]
I get an error if want to open the plugin's settings

```
nilsenpaul\cookieconsent\Plugin::getSettingsResponse(): Return value must be of type mixed, none returned
```

I am not sure when this appeared the first time. Probably after my update to Craft 4

Complete Cookie Consent 1.1
Craft 4.2.4

**Comment by nilsenpaul** [comment]
We could not reproduce this on our own install, but hopefully fixed this with the release of 1.1.1. Please upgrade and let us know!

**Comment by pimago** [comment]
Yes it's working again. Thank you!
